modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -143,10 +143,10 @@ export class {{classname}} {
 
 {{#hasQueryParams}}
         {{#useHttpClient}}
-        let queryParameters = new HttpParams({encoder: new CustomHttpUrlEncodingCodec()});
+        let queryParameters = new HttpParams({encoder: this.configuration.encoder || new CustomHttpParameterCodec()});
```

Comment:
please move this to https://github.com/OpenAPITools/openapi-generator/blob/e77b94a8b956d876f72600b9d901996aadcec235/modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache#L65-L71
to avoid accidentally missing it somewhere

Reply:
@macjohnny I additionally introduced a non-null class variable for the encoder in order to comply with strict-null-check compiler option